Spec/PushActivationStateMachine.swift

```diff
@@ -39,27 +37,37 @@ override class var defaultTestSuite : XCTestSuite {
             initialStateMachine = ARTPushActivationStateMachine(rest: rest.internal, delegate: StateMachineDelegate())
         }
 
-        afterEach {
+        func afterEach() {
```

Comment:
this one could be converted to https://developer.apple.com/documentation/xctest/xctest/1500463-teardown
>After each test completes, XCTest calls tearDown(), then tearDownWithError(), followed by tearDown(completion:). Override this method to perform any per-test cleanup.

when using it, there is no need to call this method in every test method

Reply:
Please see [this answer](https://github.com/ably/ably-cocoa/pull/1240#discussion_r760095128).